mgmtsystem_action/models/mgmtsystem_action.py

```diff
@@ -0,0 +1,185 @@
+# -*- coding: utf-8 -*-
+
+from openerp import fields, models, api
+from datetime import datetime, timedelta
+
+
+def own_company(self):
+    """Return the user company id."""
+    return self.env.user.company_id.id
+
+
+def owner(self):
+    """Return the user."""
+    return self.env.user
+
+
+class MgmtSystemAction(models.Model):
+    """Model class that manage action."""
+
+    _name = "mgmtsystem.action"
+    _description = "Action"
+    # _inherit = "crm.claim"
+    _inherit = "mail.thread"
+
+    name = fields.Char('Action Subject', required=True)
+    active = fields.Boolean('Active', default=True)
+    opening_date = fields.Datetime('Opening Date', readonly=True)
+    create_date = fields.Datetime('Creation Date', readonly=True)
+    date_deadline = fields.Date('Deadline')
+    date_closed = fields.Datetime('Closed', readonly=True)
+    date = fields.Date('Action Date', default=fields.datetime.now(),
+                       select=True)
```

Comment:
I don't know. It is a date that was take from crm_claim and i was not sure to keep it or not